save-cli/src/nativeMain/kotlin/org/cqfn/save/cli/SaveConfigReader.kt

```diff
@@ -24,13 +23,12 @@ import kotlinx.serialization.serializer
  * @return this config in case we have valid configuration
  */
 fun SaveProperties.validate(): SaveProperties {
-    val fullConfigPath = testRootPath + DIRECTORY_SEPARATOR + testConfigPath
+    this.testRootPath ?: logErrorAndExit(ExitCodes.INVALID_CONFIGURATION,
+        "`testRootPath` option is missing or null. " +
+                "Save is not able to start processing without an information about the tests that should be run.")
+    val fullConfigPath = propertiesFile!!.toPath().parent!! / testRootPath!! / testConfigPath!!
```

Comment:
Hm, compiler couldn't identify, that `testRootPath` actually couldn't be null here?

Reply:
Also, from description of `testRootPath`:
`"Path to directory with tests (relative path from place, where save.properties is stored or absolute path)"`

I think, now we don't support absolute path, right?
May be you also could change description in `config-options.json`, in aim to avoid misunderstandings

Reply:
> Hm, compiler couldn't identify, that testRootPath actually couldn't be null here?

I have actually tried it, but smart cast is unavailable here, because `testRootPath` is a mutable property

> I think, now we don't support absolute path, right?

Right, I think it could be supported relatively easy

Reply:
Okay, let's keep it as is. I opened new issue about absolute path https://github.com/cqfn/save/issues/149